packages/gatsby/src/utils/webpack-hmr-hooks-patch.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
+
+// @ts-ignore: This file assume run at browsers.
+const originalFetch = global.fetch
+// @ts-ignore
+delete global.fetch
+
+export { default } from "react-hot-loader/webpack"
```

Comment:
could we change this to a named export and update the import reference?

Reply:
might not be possible due to the way webpack works, so if not that's fine

Reply:
I tried named export, but validation of webpack config says `loader must be string`.
So I use default export.